app/api/v2/models/orders_model.py

```diff
@@ -0,0 +1,159 @@
+from db_init import db
+import datetime
+
+
+class OrderModel:
+    """blueprint for creating a parcel delivery order"""
+
+    def __init__(self, sender_name, receiver_name, receiver_contact, weight, pickup_location, destination):
+        """function to create parcel orders """
+
+        price = 50 * weight
+        self.sender_name = sender_name,
+        self.receiver_name = receiver_name,
+        self.receiver_contact = receiver_contact,
+        self.weight = weight,
+        self.pickup_location = pickup_location,
+        self.current_location = pickup_location,
+        self.destination = destination,
+        self.price = 'Ksh' + str(price),
+        self.status = "on-transit"
+        self.time_created = datetime.datetime.now()
+
+    def create_order(self):
+        """method for creating a parcel delivery order"""
+        try:
+            db.cursor.execute(
+                """
+                INSERT INTO parcels(sender_name, receiver_name, receiver_contact, weight,pickup_location,
+                current_location,destination,price,status,time_created)
+                VALUES(%s, %s, %s, %s,%s, %s, %s, %s,%s,%s) 
+                """,
+                (self.sender_name, self.receiver_name, self.receiver_contact, self.weight,
+                 self.pickup_location, self.current_location, self.destination, self.price,
+                 self.status, self.time_created)
+            )
+            db.commit()
+            return {"message": "order created successfully"}
+        except Exception as e:
+            return {"Message": e}
+
+    @staticmethod
+    def get_all_orders():
+        """ function that allows the user view all orders"""
+        db.cursor.execute("SELECT * FROM parcels ORDER BY parcel_id")
+        parcels = db.cursor.fetchall()
+        data = []
+        for k, v in enumerate(parcels):
+            parcel_id, sender_name, sender_id, receiver_name, receiver_contact, weight, pickup_location, \
+                current_location, destination, \
+                price, status, time_created = v
+            parcel = {
+                "parcel_id": parcel_id,
+                "sender_id": sender_id,
+                "sender_name": sender_name,
+                "receiver_name": receiver_name,
+                "receiver_contact": receiver_contact,
+                "weight": weight,
+                "pickup_location": pickup_location,
+                "current_location": current_location,
+                "destination": destination,
+                "price": price,
+                "status": status,
+                "time_created": time_created
+            }
+            data.append(parcel)
+        return data
+
+        return {"message": "no parcels available"}
```

Comment:
Why is this return statement here?

Reply:
It should have been indented outwards but I'll fix that. Thanks